### ISSUE TITLE
Nail psoriasis

### ISSUE THREAD

**Issue by johannaest** [obophenotype/human-phenotype-ontology#6406]
**Preferred term label:**
Nail psoriasis

**Synonyms**
Psoriatic nails

**Definition (free text, please give PubMed ID)**
Psoriasis affecting the nails.
PMID: 27041288

**Parent term (use hpo.jax.org/app)**
Psoriasis

**Comment by pnrobinson** [comment]
added
